ClosedXML_Tests/Excel/ConditionalFormats/ConditionalFormatTests.cs

```diff
@@ -21,5 +22,51 @@ public void MaintainConditionalFormattingOrder()
                 }, @"Other\StyleReferenceFiles\ConditionalFormattingOrder\ConditionalFormattingOrder.xlsx");
             }
         }
+
+        [Test]
+        public void OptionalRangeConsolidationWithoutConsolidation()
+        {
+            var options = new SaveOptions
+            {
+                ConsolidateConditionalFormatRanges = false,
+                ConsolidateDataValidationRanges = false
+            };
```

Comment:
Now it is much better. But still, this test checks too many things at once. 

So,
1. Join `OptionalRangeConsolidationWithoutConsolidation` and `OptionalRangeConsolidationWithConsolidation` in one test `ConsolidationDoesNotAffectDataBarAndColorScale`. There you don't even have to save a workbook. You can call `((XLConditionalFormats)(ws.ConditionalFormats)).Consolidate()` directly.
2. Add another test to check that `ConsolidateConditionalFormatRanges` has the effect. For this, you will create some "ordinary" conditional formats that are supposed to be consolidated with the option turned on. To avoid code duplication, create a single test with two test cases (e.g. 
```
[TestCase(true, 1)]
[TestCase(false, 2)]
public void SaveOptionAffectsConsolidation(bool consolidateConditionalFormatRanges, int expectedCount)
```
3. In `DataValidationTests` create a similar test with two test cases for conditional formats consolidation.

Reply:
Besides, once you removed those CreateAndCompare tests you also have to remove XLSX files (ConditionalFormattingOptional_consolidated.xlsx, ConditionalFormattingOptional_not_consolidated.xlsx, 
inputfile_multiple_ranges.xlsx)

Reply:
files removed

Reply:
test case implementation adjusted